tests/TestCase.php

```diff
@@ -202,7 +200,7 @@ protected function getCollectionName(): string
     {
         $class = new ReflectionClass($this);
 
-        return sprintf('%s.%s', $class->getShortName(), hash('crc32b', $this->getName()));
+        return sprintf('%s.%s', $class->getShortName(), hash('crc32b', $this->name()));
```

Comment:
I'll update to a faster algorithm
```suggestion
        return sprintf('%s.%s', $class->getShortName(), hash('xxh3', $this->name()));
```